src/workflows-app/components/TroubleshootingBox.ts

```diff
@@ -0,0 +1,99 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
```

Comment:
Instead of directly importing `colors`, suggest getting it from context:

```ts
import { useThemeFromContext } from '@terra-ui-packages/components';
...
const { colors } = useThemeFromContext();
```